### PR TITLE
Restrict profile image to an exact 110x110 size

### PR DESCRIPTION
**Pull request by staubina**

### CODE REVIEW

File: course_discovery/static/js/publisher/instructors.js [hunk 1]
```diff
@@ -126,8 +126,10 @@ function loadSelectedImage (input) {
                 reader.addEventListener("load", function (e) {
                     var image = new Image();
                     image.addEventListener("load", function () {
-                        if (image.width > imageDimension && image.height > imageDimension) {
```
Comment (Dillon-Dumesnil) [inline course_discovery/static/js/publisher/instructors.js:129]: That's weird that this used to be &&. Would that mean you could upload a 256x110 and it's valid? We are bananas

**Review by Dillon-Dumesnil — APPROVED** [review]
Woot!

File: course_discovery/apps/publisher/templates/publisher/_add_instructor_popup.html [hunk 2]
```diff
@@ -13,7 +13,7 @@ <h2 class="hd-2 emphasized new-instructor-heading">{% trans "New Instructor" %}<
                     <input name="staff-image" id="staffImageSelect" type="file" accept="image/*" onchange="loadSelectedImage(this);">
                 </div>
                 <ul>
-                    <li><span class="required"> {% trans "The image dimensions must be less than 110 x 110" %}</span>
+                    <li><span class="required"> {% trans "The image dimensions must be equal to 110x110" %}</span>
```
Comment (mikix) [inline course_discovery/apps/publisher/templates/publisher/_add_instructor_popup.html]: While we're here.... we could replace the x letter with a multiplication sign: ×

Does "must be exactly" sound any less engineer-y than "must be equal to"? Maybe not.

File: course_discovery/static/js/publisher/instructors.js [hunk 3]
```diff
@@ -126,8 +126,10 @@ function loadSelectedImage (input) {
...
-                            addModalError(gettext("The image dimensions must be less than 110 x 110"));
+                        if (image.width !== imageDimension || image.height !== imageDimension) {
+                            addModalError(
+                                gettext("The image dimensions must be equal to " + imageDimension + "x" + imageDimension)
```
Comment (mikix) [inline course_discovery/static/js/publisher/instructors.js]: Same multiplication sign note, if you're feeling typographically fancy.

**Review by mikix — APPROVED** [review]
Looks great! I've got some typography notes, but they are super ignorable.

File: course_discovery/static/js/publisher/instructors.js (hunk 1 above)
Comment (mikix) [reply course_discovery/static/js/publisher/instructors.js:129]: Yeah... That old code, not so great.

**Pull request merged**